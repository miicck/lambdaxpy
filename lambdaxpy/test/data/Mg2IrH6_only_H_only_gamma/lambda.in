500 0.25 1
1
        0.0000000000         0.0000000000         0.0000000000         0.2500000000
elph_dir/elph.inp_lambda.1
0.1