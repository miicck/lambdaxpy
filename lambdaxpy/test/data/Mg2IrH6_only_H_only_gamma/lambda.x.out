     lambda = 3.890700 (   3.894076 )  <log w>=  573.461 K  N(Ef)= 30.118728 at degauss= 0.005
     lambda = 1.954000 (   1.955669 )  <log w>=  547.331 K  N(Ef)= 17.609271 at degauss= 0.010
     lambda = 1.566600 (   1.567926 )  <log w>=  533.335 K  N(Ef)= 15.385715 at degauss= 0.015
     lambda = 1.522200 (   1.523489 )  <log w>=  533.440 K  N(Ef)= 14.030434 at degauss= 0.020
     lambda = 1.421000 (   1.422206 )  <log w>=  535.520 K  N(Ef)= 12.586812 at degauss= 0.025
     lambda = 1.283000 (   1.284090 )  <log w>=  537.177 K  N(Ef)= 11.342137 at degauss= 0.030
     lambda = 1.145000 (   1.145972 )  <log w>=  539.239 K  N(Ef)= 10.345183 at degauss= 0.035
     lambda = 1.022500 (   1.023365 )  <log w>=  542.087 K  N(Ef)=  9.556968 at degauss= 0.040
     lambda = 0.920100 (   0.920872 )  <log w>=  545.978 K  N(Ef)=  8.932110 at degauss= 0.045
     lambda = 0.836800 (   0.837490 )  <log w>=  551.691 K  N(Ef)=  8.434582 at degauss= 0.050
lambda        omega_log          T_c
   3.89070       573.461            114.022
   1.95400       547.331             77.466
   1.56660       533.335             63.289
   1.52220       533.440             61.654
   1.42100       535.520             57.899
   1.28300       537.177             52.054
   1.14500       539.239             45.491
   1.02250       542.087             39.046
   0.92010       545.978             33.224
   0.83680       551.691             28.265
